### DISCUSSION TITLE
Welcome to graphql-zeus Discussions!

### DISCUSSION THREAD

**Discussion by aexol** [graphql-editor/graphql-zeus#263] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by seanonthenet** [comment]
Awesome! Thanks @aexol 
I'm super passionate about GraphQL Zeus. It has changed the way I build projects completely. I mainly use it with self hosted Hasura and it handles my huge schemas and generates usually in less than a second. :)

I recently designed a new logo, rewrote the `README.md` and the Zeus documentation for the GraphQL Editor team and want to continue doing that to make Zeus even easier and seamless for newcomers. 

I'd also like to be contributing more to the core and am currently skilling up my advanced Typescript to be able to help with that. (If there were more detailed guidelines of _how_ Zeus actually works to build schemas, and how to setup quick Zeus core dev environments that would be really good. 🚀 )

I'm also keen to start using Zeus with other public GraphQL APIs like Shopify which I use a lot. 

I'm keen to help anyone who is starting out with Zeus. 🤝

**Comment by aexol** [comment]
That's why I pushed 5.0 which should be understandable and easier to maintain. All the functions are now in TS and there is a produce-lib script that generates strings for them so the development of this package is much easier. Maybe we should have a call or discuss here the future of zeus.

**Reply by seanonthenet** [reply]
I'm very keen to discuss the future of Zeus :)

I think opening a Discussion here on the new forum, maybe in its own thread, is the best way so that it's out in the open and everyone can see and contribute if they want

To start off the discussion, I think there are some questions we can put down to consider:

• How can we attract more users to Zeus?
• How can we attract more maintainers and contributors? (Functions to TS etc is great) 
• How can we formalise and improve releases and changes?
• How can we improve the documentation to be as clear and descriptive as possible and help more people?
• How can Zeus help the general GraphQL userbase and ecosystem?
• Are there other things that Zeus could power in the GraphQL ecosystem? Eg: A dedicated official hooks based Zeus React client?
• How can Zeus better serve its "parent" entity (GraphQL Editor)?

So that's a start. Let me know if you want to me add some discussion topics for each of these or if you have more?

Thanks
Sean
